packages/react/src/useEntityQuery.ts

```diff
@@ -7,19 +7,31 @@ import { distinctUntilChanged, map } from "rxjs";
 // This does a little more rendering than is necessary when arguments change,
 // but at least it's giving correct results now. Will optimize later!
 
-export function useEntityQuery(fragments: QueryFragment[]) {
+/**
+ * Returns all matching `EntityIndex`es for a given entity query,
+ * and triggers a re-render as new query results come in.
+ *
+ * @param fragments Query fragments to match against, executed from left to right.
+ * @param options.recomputeOnValueChange False - re-renders only on entity array changes. True (default) - also on component value changes.
+ * @returns Set of entities matching the query fragments.
+ */
+export function useEntityQuery(fragments: QueryFragment[], options?: { recomputeOnValueChange?: boolean }) {
+  const recomputeOnValueChange = options?.recomputeOnValueChange ?? true;
```

Comment:
This looks great, thank you!

Not blocking but I'm wondering if something like `updateOnValueChange` would align a little better with existing React terminology. And I'm also wondering if we should invert this and have the option be something like "update on distinct entities" or "ignore component value changes", but that's a mouthful.

Reply:
update does seem better, changed it